core/src/test/java/io/questdb/cutlass/http/DispatcherWriterQueueTest.java

```diff
@@ -462,8 +462,8 @@ private void runAlterOnBusyTable(
                 AtomicInteger errors = new AtomicInteger();
                 CyclicBarrier barrier = new CyclicBarrier(httpAlterQueries.length);
 
-                for (int i = 0; i < httpAlterQueries.length; i++) {
-                    String httpAlterQuery = httpAlterQueries[i].replace("<x>", tableName);
+                for (String alterQuery : httpAlterQueries) {
```

Comment:
nit: I appreciate this is a test, but we should have one standard to iterate an array rather than many. Lets stick with our usual method, which we wouldn't hesitate to use in production code

Reply:
GH malfunctions, there should be only one comment

Reply:
Reverted in d23bcf54bd699edcad89e085c39ea9e41acc3ba5

As for array iteration, our approach is no different than the enhanced for loop. This gist compares the output byte code of both approaches:
https://gist.github.com/puzpuzpuz/104edaf8ba1a875a6f1823a790d545e0